homeassistant/components/supla/__init__.py

```diff
@@ -44,7 +46,10 @@
 CONFIG_SCHEMA = vol.Schema(
     {
         DOMAIN: vol.Schema(
-            {vol.Required(CONF_SERVERS): vol.All(cv.ensure_list, [SERVER_CONFIG])}
+            {
+                vol.Required(CONF_SERVERS): vol.All(cv.ensure_list, [SERVER_CONFIG]),
+                vol.Optional(CONF_NO_CHECK_CERTIFICATE): cv.boolean,
```

Comment:
We no longer accept additions or changes to the yaml config of integrations. See: https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md

This integration first needs to be refactored to use a config flow and config entries.

https://developers.home-assistant.io/docs/config_entries_config_flow_handler